.github/workflows/windows.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     - name: test library
       shell: bash
       run: |
-        cmake --build build  --target test -- -m
+        cmake --build build  --target test -- -m:1
```

Comment:
I am still baffled that we apparently run into a concurrency issue with the quick tests here.

Reply:
Yep.

Reply:
Will clean that up first, so please wait with merging